app/_locales/en/messages.json

```diff
@@ -2966,11 +2966,14 @@
     "message": "Install Snap"
   },
   "snapInstallWarningCheck": {
-    "message": "To confirm you understand, check all."
+    "message": "To confirm you understand, check the box."
+  },
+  "snapInstallWarningCheckPlural": {
+    "message": "To confirm you understand, check all the boxes."
   },
   "snapInstallWarningKeyAccess": {
-    "message": "You are granting key access to the snap \"$1\". This is irrevocable and grants \"$1\" control of your accounts and assets. Make sure you trust \"$1\" before proceeding.",
-    "description": "The parameter is the name of the snap"
+    "message": "You are granting $2 key access to the snap \"$1\". This is irrevocable and grants \"$1\" control of your $2 accounts and assets. Make sure you trust \"$1\" before proceeding.",
+    "description": "The first parameter is the name of the snap and the second one is the protocol"
```

Comment:
Do we need to remove the localized versions of this message since you've changed the parameters? Otherwise the localized messages will be outdated.